src/components/sbb-notification/sbb-notification.scss

```diff
@@ -0,0 +1,216 @@
+@use '../../global/styles' as sbb;
+@use 'sass:color';
+@use 'node_modules/@sbb-esta/lyne-design-tokens/dist/scss/sbb-variables.scss' as sbb-tokens;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-notification-visibility: hidden;
+  --sbb-notification-opacity: 0;
+  --sbb-notification-max-height: 0;
+  --sbb-notification-margin: 0;
+  --sbb-notification-padding-block: var(--sbb-spacing-responsive-xxs);
+  --sbb-notification-padding-inline: var(--sbb-spacing-responsive-xs);
+  --sbb-notification-color: var(--sbb-color-charcoal-default);
+  --sbb-notification-type-color: var(--sbb-color-granite-default);
+  --sbb-notification-type-color-sass: #{color.mix(sbb-tokens.$sbb-color-granite-default, white, 5%)};
+  --sbb-notification-border-width: var(--sbb-border-width-1x);
+  --sbb-notification-border: var(--sbb-border-width-1x) solid var(--sbb-notification-type-color);
+  --sbb-notification-base-radius: var(--sbb-border-radius-4x);
+  --sbb-notification-border-radius: calc(var(--sbb-notification-base-radius) / 2)
+    var(--sbb-notification-base-radius) var(--sbb-notification-base-radius)
+    calc(var(--sbb-notification-base-radius) / 2);
+  --sbb-notification-animation-duration: var(--sbb-animation-duration-4x);
+  --sbb-notification-timing-function: ease-in;
+  --sbb-notification-transition: max-height var(--sbb-notification-animation-duration)
+      var(--sbb-animation-duration-2x) var(--sbb-notification-timing-function),
+    padding var(--sbb-notification-animation-duration) var(--sbb-animation-duration-2x)
+      var(--sbb-notification-timing-function),
+    border var(--sbb-notification-animation-duration) var(--sbb-animation-duration-2x)
+      var(--sbb-notification-timing-function),
+    visibility var(--sbb-notification-animation-duration) var(--sbb-notification-timing-function),
+    opacity var(--sbb-notification-animation-duration) var(--sbb-notification-timing-function);
+
+  margin: 0;
+  transition: margin var(--sbb-notification-animation-duration)
+    var(--sbb-notification-timing-function);
+}
+
+:host(:is([data-state='opened'], [data-state='opening'])) {
+  --sbb-notification-visibility: visible;
+  --sbb-notification-opacity: 1;
+  --sbb-notification-max-height: calc(
+    var(--sbb-notification-height) + (var(--sbb-notification-border-width) * 2)
+  );
+  --sbb-notification-transition: max-height var(--sbb-notification-animation-duration)
+      var(--sbb-notification-timing-function),
+    padding var(--sbb-notification-animation-duration) var(--sbb-notification-timing-function),
+    border var(--sbb-notification-animation-duration) var(--sbb-notification-timing-function),
+    opacity var(--sbb-notification-animation-duration) var(--sbb-notification-animation-duration)
+      var(--sbb-notification-timing-function);
+
+  margin: var(--sbb-notification-margin);
```

Comment:
I just discovered this "hack", is it because of the animation? I think, if not otherwise possible, we should describe this css var in documentation. But e.g. the gap case would then not really be possible :/.. hmmm